guiEditor/src/stringTools.ts

```diff
@@ -0,0 +1,97 @@
+import { NodeMaterialBlockConnectionPointTypes } from 'babylonjs/Materials/Node/Enums/nodeMaterialBlockConnectionPointTypes';

```

Comment:
That should be shared

Reply:
Just removed since it's actually not being used.